src/main/kotlin/club/sk1er/mods/core/universal/wrappers/message/UTextComponent.kt

```diff
@@ -444,13 +444,20 @@ class UTextComponent : MCITextComponent {
 
     override fun getFormattedText(): String = component.formattedText
 
-    override fun getSiblings(): MutableList<IChatComponent> = component.siblings
-
     override fun createCopy(): IChatComponent = component.createCopy()
 
+    //#if MC<=10800
```

Comment:
Is this supposed to work on just 1.7.10, or for both 1.7.10 and 1.8.9? It should be `10710` or `10809`, respectively.